app/models/user.rb

```diff
@@ -1,7 +1,24 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id              :integer          not null, primary key
+#  name            :string
+#  email           :string
+#  password_digest :string
+#  project_id      :string
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  session_token   :string
+#
+
 class User < ApplicationRecord
   has_secure_password
   before_save { email.downcase! }
 
+  has_many :planets, dependent: :destroy
+  has_many :assignments, through: :planets, dependent: :destroy
+
```

Comment:
これ、userはprojectを複数持ってるはずよね。
やから、こういうアソシエーションが必要なんやないかな
```suggestion
has_many :projects, throgh: :user_projects
```

Reply:
これどうする？

Reply:
has_many:throughはhas_and_belongs_to_manyで代用しました